src/Experimental/degeneracy.jl

```diff
@@ -0,0 +1,99 @@
+"""
+    PKC(g[, frac = 0.95])
```

Comment:
I think we should rename this to `parallel_k_core()`. And we also need to figure out a better way to handle parallel equivalents of methods.

Also, you should document `frac` - what's its significance?

Reply:
scratch that. The new convention is `threaded_k_core()`.